cis194/week2/zehua/src/Exercises.hs

```diff
@@ -0,0 +1,23 @@
+module Exercises where
+
+import Log
+
+-- ex1
+createLogMessage :: MessageType -> String -> [String] -> LogMessage
+createLogMessage mt ts msg = LogMessage mt (read ts :: TimeStamp) (unwords msg)
```

Comment:
Consider edge cases, what if `ts` is `"a"`? Please try some of the `readXXX` functions instead of `read` to avoid runtime exceptions.

Reply:
yep. fixed.